app/src/main/res/xml/preferences.xml

```diff
@@ -175,4 +175,8 @@
             android:summary="@string/restart_desc"
             android:title="@string/restart_name" />
     </PreferenceScreen>
+    <Preference
+        android:key="rate-app"
+        android:title="Rate the app">
```

Comment:
I'm not convinced we need this "feature" at at, but what ever :wink:

Please using a string reference (`@string/rate_app_name`) here so that this can be translated to other languages (like french).